cmd/gardener-extension-networking-calico/app/app.go

```diff
@@ -75,7 +77,13 @@ func NewControllerManagerCommand(ctx context.Context) *cobra.Command {
 			}
 			util.ApplyClientConnectionConfigurationToRESTConfig(configFileOpts.Completed().Config.ClientConnection, restOpts.Completed().Config)
 
-			mgr, err := manager.New(restOpts.Completed().Config, mgrOpts.Completed().Options())
+			completedMgrOpts := mgrOpts.Completed().Options()
+			completedMgrOpts.NewClient = utils.NewClientFuncWithDisabledCacheFor(
+				&corev1.Secret{},    // applied for ManagedResources
+				&corev1.ConfigMap{}, // applied for monitoring config
+			)
+
+			mgr, err := manager.New(restOpts.Completed().Config, completedMgrOpts)
```

Comment:
@timuthy WDYT about the usage shown here?
Do you think it makes sense, to make this somehow configurable in `controllercmd.ManagerOptions`?

Reply:
For now it is sufficient and simple, no need to make it configurable for operators, imo. We can still do it if it turns out to be necessary.